packages/file/lib/src/backends/memory/memory_file_system.dart

```diff
@@ -4,6 +4,7 @@
 
 import 'package:file/file.dart';
 import 'package:file/src/backends/memory/operations.dart';
+import 'package:file/src/io_overrides.dart';
```

Comment:
Does this library today have a transitive import to `dart:io`? I would think it doesn't, if the point is to be an in memory file system (for use where `dart:io` isn't available).

This IOOverrides functionality probably needs to be exposed through a separate library to avoid polluting this one with that import.

Reply:
Yes that is correct, I don't think I was aware of how this package works when I made that comment (and also.... it really shouldn't work that way???).